transformations/GermanGenderSwap/test.json

```diff
@@ -0,0 +1,50 @@
+{
+  "type": "GermanGenderSwap",
+  "test_cases": [
+    {
+      "class": "GermanGenderSwap",
+      "inputs": {
+        "sentence": "Mein Vater und dein Bruder sind hier?"
+      },
+      "outputs": [{
+        "sentence": "Meine Mutter und deine Schwester sind hier?"
+      }]
+    },
+    {
+      "class": "GermanGenderSwap",
+      "inputs": {
+        "sentence": "Er ist ein Arzt und mein Vater."
+      },
+      "outputs": [{
+        "sentence": "Sie ist eine Arztin und meine Mutter."
+      }]
+    },
+    {
+      "class": "GermanGenderSwap",
+      "inputs": {
+        "sentence": "Ich sehe den Mann."
+      },
+      "outputs": [{
+        "sentence": "Ich sehe die Frau."
+      }]
+    },
+    {
+      "class": "GermanGenderSwap",
+      "inputs": {
+        "sentence": "Er ist der Prasident, sie ist die Herrscherin und sie werden jetzt bald Ehemann und Ehefrau sein!"
+      },
+      "outputs": [{
+        "sentence": "Sie ist die Prasidentin, sie ist die Herrscherin und sie werden jetzt bald Frau und Ehefrau sein!"
```

Comment:
Same umlaut error as before

Reply:
Here, the umlauts are now displayed as Unicode, however, if the resulting json file is opened separately, or displayed in a Notebook cell, they can be read in their original form.